docs/migration/awscloudwatch/fluentd.md

```diff
@@ -0,0 +1,119 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Fluentd
+
+*Migrating/Stream Logs from AWS CloudWatch to SigLens through Fluentd.*
+
+
+**Prerequisites**: AWS account with logs in CloudWatch Logs Group.
+
+
+## 1. Fluentd Installation
+
+Install [Fluentd](https://docs.fluentd.org/installation) on your server.
+
+### 2. IAM Role Configuration
+
+1. Create an IAM role with the below permissions.
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Action": [
+                    "logs:GetLogEvents",
+                    "logs:DescribeLogStreams"
+                ],
+                "Effect": "Allow",
+                "Resource": "*"
+            }
+        ]
+    }
+    ```
+2. Obtain IAM Role Access Key and Secret Key for authentication.
+3. This is required for the Authentication purposes.
```

Comment:
It looks like 3 should be part of 2. Also a little more guidance on how to get the Access Key and Secret Key would be good